TKGame/UI/DebugMenu.cs

```diff
@@ -93,7 +93,7 @@ private static void Initialize()
         /// </summary>
         private static void LoadContent()
         {
-            VSP.Margin = new Myra.Graphics2D.Thickness(100, 100, 0, 0);
+            VSP.Margin = new Myra.Graphics2D.Thickness(100, 25, 0, 0);
```

Comment:
Should probably add comments here to specify what the second argument is and why it is changed. I assumed its just margin sizing?